internal/clusterconfig/clusterconfig_test.go

```diff
@@ -0,0 +1,125 @@
+package clusterconfig_test
+
+import (
+	"context"
+
+	"github.com/kyma-project/istio/operator/internal/clusterconfig"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	operatorv1alpha1 "github.com/kyma-project/istio/operator/api/v1alpha1"
+
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var _ = Describe("EvaluateClusterConfiguration", func() {
+	Context("k3d", func() {
+		It("should set cni values and serviceAnnotations to k3d configuration", func() {
+			//given
+			k3dNode := corev1.Node{
+				ObjectMeta: v1.ObjectMeta{
+					Name: "k3d-node-1",
+				},
+				Status: corev1.NodeStatus{
+					NodeInfo: corev1.NodeSystemInfo{
+						KubeProxyVersion: "v1.25.6+k3s1",
```

Comment:
Wouldn't it be better to have this value as a variable/constant with explanatory name? For example `KubeProxyVersionK3S`.
But since the whole object is named properly it also can be accepted.